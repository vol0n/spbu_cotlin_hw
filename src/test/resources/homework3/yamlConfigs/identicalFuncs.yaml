package name: identicalFuncs
class name: IdenticalFuncs
functions:
  - name: "backwardApply"
  - name: "backwardApply"