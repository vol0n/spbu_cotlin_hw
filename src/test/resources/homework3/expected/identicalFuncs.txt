package identicalFuncs

import kotlin.Unit
import org.junit.jupiter.api.Test

public class IdenticalFuncsTest {
  @Test
  public fun backwardApply(): Unit {
  }
}
